tests/production/Common.js

```diff
@@ -0,0 +1,143 @@
+    gpii.tests.productionConfigTesting.getKeyOrPrefsFromFile(
+        "%gpii-universal/build/tests/dbData/prefsSafes.json",
+        "prefsSafe-os_gnome"
+    );
+
+gpii.tests.productionConfigTesting.prefsUpdate = {
+    "contexts": {
+        "gpii-default": {
+            "name": "Default preferences",
+            "preferences": {
+                "http://registry.gpii.net/common/setting1": 12
+            }
+        }
+    }
+};
+
+gpii.tests.productionConfigTesting.testStatusCode = function (data, request) {
+    jqUnit.assertEquals(
+        "Checking status of " + request.options.path,
+        request.options.expectedStatusCode, request.nativeResponse.statusCode
+    );
+};
+
+gpii.tests.productionConfigTesting.testAddedToDatabase = function (data, request) {
+    var expected = request.options.expectedStatusCodes;
+    var actual = request.nativeResponse.statusCode;
+    jqUnit.assertNotEquals(
+        "Adding record to database using " + request.options.path +
+        ", status: " + actual,
+        expected.indexOf(actual), -1
+    );
+    // Store the added record's id and rev
+    if (actual === 201 || actual === 200) {
+        request.options.result = JSON.parse(data);
+        fluid.log(request.options.result);
+    }
+};
+
+gpii.tests.productionConfigTesting.testResponse = function (data, request) {
+    gpii.tests.productionConfigTesting.testStatusCode(data, request);
+    jqUnit.assertDeepEq(
+        "Checking paylod of " + request.options.path,
+        request.options.expectedPayload, JSON.parse(data)
+    );
+};
+
+gpii.tests.productionConfigTesting.testAccessResponse = function (data, request) {
+    var token = JSON.parse(data);
+    var auth = "Bearer " + token.access_token;
+    request.options.stashedAuth = auth;
+
+    gpii.tests.productionConfigTesting.testStatusCode(data, request);
+    jqUnit.assertNotNull("Checking 'access_token'", token.access_token);
+    jqUnit.assertNotNull("Checking 'expiresIn'", token.expiresIn);
+    jqUnit.assertEquals("Checking 'token_type'",  "Bearer", token.token_type);
+};
+
+gpii.tests.productionConfigTesting.testLifecycleResponse = function (data, request) {
+    gpii.tests.productionConfigTesting.testStatusCode(data, request);
+
+    var lifeCycle = JSON.parse(data);
+    jqUnit.assertEquals(
+        "Checking lifeCycle user '" + gpii.tests.productionConfigTesting.carlaKey + "'",
+        gpii.tests.productionConfigTesting.carlaKey,
+        lifeCycle.gpiiKey
+    );
+    // These checks based on
+    // https://github.com/GPII/universal/blob/master/documentation/FlowManager.md#get-lifecycle-instructions-from-cloud-based-flow-manager-get-gpiikeysettingsdevice
+    jqUnit.assertNotNull("Checking 'solutionsRegistryEntries'", lifeCycle.solutionsRegistryEntries);
+    jqUnit.assertNotNull("Checking 'matchMakerOutput'", lifeCycle.matchMakerOutput);
+};
+
+gpii.tests.productionConfigTesting.testGetForDeletion = function (data, request) {
```

Comment:
Can we rename this function to something like `saveDocForDeletion`?

Reply:
How about `testGetAndSavDocForDeletion()`?  The pattern I've been trying to use (not always following through, though) is to begin the name of the testing function "test" and then name it according to what it is testing.  So here, it is testing that the GET was successful.  Then, if successful stash it away for a subsequent deletion.

Reply:
That's a good name. Let's go with it. Thanks.